test/alternative_spec.rb

```diff
@@ -0,0 +1,51 @@
+require_relative "spec_helper.rb"
+
+describe "UpdateAlternatives::Alternative" do
```

Comment:
I miss require call above and then here can be used `describe UpdateAlternative do` ( so no string, passing object instead )